### ISSUE TITLE
Add missing audits

### ISSUE THREAD

**Issue by drorsnir** [nischayn22/PageQuality#20]
- [x] list_items_per_level
- [x] "Callouts":
  - [x] callouts_adjacent
  - [x] callouts_at_section_start

**Comment by nischayn22** [comment]
Lists should not have many same-level bullets - they should have sub-bullets, or divided, to maintain readability - Can you help with a clearer rule. Should I consider the number of consecutive same-level bullets without any sub-bullets?

**Comment by nischayn22** [comment]
We should not have 2 callouts, one after the other - Do you mean there should be some other HTML tag and content between 2 callouts or is it more about the actual number of characters between two callouts?

No callouts should precede the section text - Can you help me with an example please.

**Comment by drorsnir** [comment]
> Lists should not have many same-level bullets - they should have sub-bullets

Exactly, for every `<ul>` or `<ol>` you should just look at the number of child `<li>` tags. it should be no more then `list_items_per_level`.

> We should not have 2 callouts, one after the other
> Do you mean there should be some other HTML tag and content

Exactly. Just something should be there (besides a `<br>` and whitespace

> No callouts should precede the section text - Can you help me with an example please.
```
<h2> Hello </h2>
<div class="wr-example">lorem ipsum</div>
```
So there's no text between the section headline and the callout.

**Comment by nischayn22** [comment]
https://github.com/nischayn22/PageQuality/commit/84ace3b3781b06488ba758a11c46041b4465a70b - list items per level

**Comment by nischayn22** [comment]
https://github.com/nischayn22/PageQuality/commit/e830c57f1b109c1b7d77af8c5ca6676fa68321df - callout at section start

**Comment by nischayn22** [comment]
https://github.com/nischayn22/PageQuality/commit/7d721eead0f03672d3b20e2e1f8012ced49f6b89 - callouts adjacent to each other

**Comment by nischayn22** [comment]
@FreedomFighterSparrow Please review and close this.
